apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -59,6 +59,14 @@ type ContourSettings struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// KubernetesDebug Enable Kubernetes client debug logging with log level. If unset,
+	// defaults to 0.
+	//
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=9
+	// +optional
+	KubernetesDebug uint8 `json:"kubernetesDebug,omitempty"`
```

Comment:
would prefer a rename to `KubernetesLogLevel` to make it a little clearer
```suggestion
	KubernetesLogLevel uint8 `json:"kubernetesDebug,omitempty"`
```

Reply:
just to let it match with
```
serve.Flag("kubernetes-debug", "Enable Kubernetes client debug logging with log level.").PlaceHolder("<log level>").UintVar(&ctx.KubernetesDebug)
```
if you think renaming it is better, I will change it

Reply:
yeah I would prefer the rename: personally don't love the existing flag name and I would like to new field to be a bit more readable

no need to match the field name exactly with the flag, as people using the provisioner don't need to know the exact flag name, we can make it configurable with something more clear what the meaning is

Reply:
done